clients/vault/Cargo.toml

```diff
@@ -6,7 +6,9 @@ name = "vault"
 version = "0.0.1"
 
 [features]
-integration = []
+integration = [
+    "wallet/testing-utils"
```

Comment:
see `fn listen_for_new_transactions(...)` in _clients/wallet/src/horizon/horizon.rs_